firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -182,9 +182,15 @@ FetchResponse fetch(
       String lastFetchETag,
       Map<String, String> customHeaders,
       Long firstOpenTime,
-      Date currentTime)
+      Date currentTime,
+      Boolean excludeEtagHeaderForRealtime)
```

Comment:
same thing since this is downstream of the fetch handler... I think this can just be `excludeEtagHeader`